### ISSUE TITLE
Failing to reference existing S3 Bucket while creating lambda layer

### ISSUE THREAD

**Issue by texalbuja** [aws-ia/terraform-aws-control_tower_account_factory#18]
We are following the tutorial 

https://learn.hashicorp.com/tutorials/terraform/aws-control-tower-aft#create-aws-aft-organizational-unit-and-account

We are logged in a non-root account with Administrator Access, inside a brand new account with Control Tower and AFT OU.

We are facing the following error while provisioning AFT account.

![image](https://user-images.githubusercontent.com/6930415/147155225-d1baf927-71b6-4415-92e7-a869a5ef49a0.png)

After looking inside the lambda module, we find that the bucket is already created.

![image](https://user-images.githubusercontent.com/6930415/147155725-8aa54b94-641e-4835-b84a-0d3aec20be42.png)

We already destroyed and recreated AFT but the problem persist.

Would you like to take a look?

**Comment by balltrev** [comment]
Can you check the output of the latest build run for CodeBuild project "python-layer-builder-aft-common-*"? This failure could be due to the lambda layer object not existing in S3, which is delivered by the CodeBuild project I mentioned before.

**Comment by texalbuja** [comment]
Thank for reaching out @balltrev, quick update, I tried to run the build and it looks like the aft management account didn't have enough quota for run builds. After getting in touch with aws support, they enabled the account to run that build and the lambda layer object was created. Finally the terraform apply worked as expected. I'm guessing that the root problem is that I just created this brand new account.
